test/services/user/services/AdminUsers.test.js

```diff
@@ -848,12 +849,12 @@ describe('AdminUsersService', () => {
 
 		const deletedStudent = await adminStudentsService.remove(null, params);
 		expect(deletedStudent).to.not.be.undefined;
-		expect(deletedStudent.firstName).to.equals('validDeleteStudent');
+		expect(deletedStudent.length).to.equals(1);
```

Comment:
lengh test are dangerous if the right user is deleted? It can only validate over the _id of the user